config/routes.rb

```diff
@@ -61,6 +61,7 @@
           get tab, to: "activities#show", defaults: {tab: tab}
         end
       end
+      resources :level_b_activity_uploads, only: [:new]
```

Comment:
Shall we make this `resource` here rather than `resources` as (as you so eloquently put it in a recent commit message):
> these routes aren't here to perform actions on database records of uploads (with upload IDs), but to upload activity data relating to organisations